azure-pipelines.yml

```diff
@@ -474,7 +474,7 @@ stages:
                 SQ_VERSION: "LATEST_RELEASE[8.9]"
                 SONAR_DOTNET_VERSION: '9.7.0.75501' # sonar-dotnet 9.8 will mention SQ 9.9 as minimum supported version
                 SONAR_CFAMILYPLUGIN_VERSION: "6.20.5.49286"  # LATEST_RELEASE of CFAMILY is not compatible with old SQ
-                MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\MSBuild\\Current\\Bin\\MSBuild.exe"
+                MSBUILD_PATH: "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\MSBuild\\Current\\Bin\\MSBuild.exe"
```

Comment:
Do we have to switch to community? I've noticed that the build tools are also installed: https://github.com/SonarSource/re-ci-images/blob/master/ec2-images/ansible/roles/tools/tasks/visualstudio.yml#L2-L6